config_custom/control.ini

```diff
@@ -1,23 +1,9 @@
 [Control]
 exclude = pretest_example,intratest_example,posttest_example,example,subexample
-          ,docker_cli/build/git_path
-          ,docker_cli/build/dockerignore
-          ,docker_cli/cp
-          ,docker_cli/info
-          ,docker_cli/build
-          ,docker_cli/deferred_deletion
-          ,docker_cli/events
-          ,docker_cli/images_all
-          ,docker_cli/liverestore
-          ,docker_cli/load
-          ,docker_cli/login
-          ,docker_cli/pull
-          ,docker_cli/run_attach
-          ,docker_cli/run_cgroups
-          ,docker_cli/run_exec/exec_false
-          ,docker_cli/run_exec/exec_pid_count
-          ,docker_cli/run_sigproxy
-          ,docker_cli/selinux_labels
-          ,docker_cli/stop/stopped
-          ,docker_cli/systemd
-          ,redhat/rhel_push_plugin
+subtests = docker_cli/create,
+           docker_cli/dockerhelp,
+           docker_cli/import_export,
+           docker_cli/run,
+           docker_cli/start,
+           docker_cli/stop,
+           docker_cli/version,
```

Comment:
The version test is useless for podman; I see no reason to include it.

Reply:
Some that I think worth including: attach, commit, run_cgroup_parent, build, kill_stopped, logs, wait, run_cidfile, kill, kill_bad, rm, history